### ISSUE TITLE
package-lock.json should also be included in .editorconfig

### ISSUE THREAD

**Issue by aashutoshrathi** [fossasia/susi_desktop#85]
**Actual Behaviour**

.editorconfig has only package.json in json files

**Steps to reproduce it**

We better add *.json for checking all JSON files.

##### Please check the applicable fields -

- [x] I ensure that this issue is not a duplicate of any current open issues(s).
- [x] Assign this issue to me (I want to work on this).
- [x] I will send my PR only after I'm assigned to this issue by any maintainer.
- [x] I understand that I will be unassigned from this issue if I fail to send a PR for this within a week.

**Comment by anshumanv** [comment]
GFI :+1: 
Remove Headings which are not applicable.
